kolibri/content/api.py

```diff
@@ -24,6 +26,28 @@ class ChannelMetadataCacheViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return models.ChannelMetadataCache.objects.all()
 
+    def destroy(self, request, pk=None):
```

Comment:
Does the permission policy apply to the whole viewset? It seems you can attach it to the whole thing, but that would restrict GET, POST, as well.